exchanges/poloniex/poloniex_websocket.go

```diff
@@ -948,11 +948,13 @@ func (p *Poloniex) processAccountBalanceUpdate(notification []any) error {
 	// NOTES: This will affect free amount, a rest call might be needed to get
 	// locked and total amounts periodically.
 	p.Websocket.DataHandler <- account.Change{
```

Comment:
needs to use `account.ProcessChange` and take in the slice of account.Change please 🙏

Reply:
It looks like Poloniex is using the WebSocket endpoint "wss://api2.poloniex.com", but it seems this WebSocket is no longer maintained. Do we still need to support it? Parsing the data could be an issue.

Reply:
Leave as is 👍